launch_debug.py

```diff
@@ -6,6 +6,8 @@
 dock_widget, plugin_widget = viewer.window.add_plugin_dock_widget(
     "cavendish-particle-tracks",
 )
+
+viewer.window.add_dock_widget(dock_widget, area="bottom")
```

Comment:
Oh nice! Can we make a launcher for the installations in the lab, maybe?

It'd be even nicer if we could have this as a property of the plugin somehow... Like whenever I start `napari` I open the plugin and it's there. I will dig in the plugin docs.